client/src/components/_common/SectionContent/SectionContent.layouts.module.css

```diff
@@ -61,7 +61,7 @@
   .multi-column {
     column-gap: var(--column-gap);
     column-rule: 1px solid rgb(112 112 112 / 25%);
-    column-fill: auto;
+    column-fill: balance;
```

Comment:
**Unintended Affect on Core?**
This seems to affect Core… layout of Dashboard changes.

<details>

![Core Layout Changed](https://user-images.githubusercontent.com/62723358/186703204-2f062611-261d-4a46-93e8-fb183f6bcf66.png)

</details>

Reply:
Maybe there is a way to change the layout only if `hideApps` is set. Like a conditionally-loaded classname or stylesheet… then a property for SectionContent is needed... crap. I'll try to come up with something slim from 11:00–13:00.

Reply:
Unintended. I wonder why it's filling in right to left. It sounded like `balance` filled left to right.

Reply:
I don't know what is filling in right to left. Your code has panels fill in left to right to me. And it does so whether the CSS is set to `balance` or `auto` (or the `contentLayoutName` is set to `balance` or `twoColumn`).

Reply:
Balance does something. I was testing my first off-branch after merging in yours, and I see it has an effect. And I understand why you made `balanceUnequal`. You're great.